cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -390,12 +442,22 @@ static DataprocConf create(Map<String, String> properties, @Nullable SSHPublicKe
     String gcpCmekKeyName = getString(properties, "encryptionKeyName");
     String gcpCmekBucket = getString(properties, "gcsBucket");
 
+    String clustermetadata = getString(properties, "clustermetadata");
```

Comment:
Should be camel case `clusterMetadata`. Same for the string value in the property

Reply:
Done